sdk/eventhub/arm-eventhub/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Release History
     
+## 5.3.0-beta.1 (2025-02-06)
+Compared with version 5.2.0
+    
+### Features Added
+
+  - Added operation Namespaces.beginFailover
+  - Added operation Namespaces.beginFailoverAndWait
+  - Added Interface ErrorDetailAutoGenerated
```

Comment:
why we have auto generated?

Reply:
pending: https://github.com/Azure/sdk-release-request/issues/5803#issuecomment-2642263488